cookiecutter.json

```diff
@@ -1,5 +1,9 @@
 {
     "project_name": "sugardough",
+    "name": "Sugardough",
+    "project_description": "Add your project description",
     "project_db": "sugardough_db",
-    "project_author": "Mozilla Foundation"
+    "project_author": "Mozilla Foundation",
+    "license": "Apache 2.0",
```

Comment:
Actually we don't need this as a cookiecutter variable. Sugardough based projects are by default MPL2, we even include the license in the repo. If one wants to change this, (s)he has to do more than just answering the cookiecutter question. 

So I suggest that you remove this and you hardcode `MPL 2.0` in `contribute.json`